pulsar-functions/proto/src/main/proto/Function.proto

```diff
@@ -69,6 +69,7 @@ message SourceSpec {
     // configs used only when source feeds into functions
     SubscriptionType subscriptionType = 3;
     map<string,string> topicsToSerDeClassName = 4;
+    uint64 timeoutMs = 6;
```

Comment:
Do we want to call it pulsar specifically since its only applicable to pulsar topics?

Reply:
Everything field under the comment: 

"// configs used only when source feeds into functions"

are pulsar source specific configs.  We didn't name them pulsar source specific thus, I don't think we should that it for timeoutMs